cu-calendar.sty

```diff
@@ -112,13 +112,13 @@
 }%
 %
 \def\cuDayName#1{%
-  \ifcase#1 воскресение\or%
-    понедельник\or
-    вторник\or
-    среда\or
-    четверг\or
-    пятница\or
-    суббота\fi
+  \ifcase#1 недѣ́лѧ\or%
+    понедѣ́льникъ\or
+    вто́рникъ\or
+    среда̀\or
+    четверто́къ\or
+    пѧто́къ\or
+    сꙋббѡ́та\fi
 }%
```

Comment:
It seems an overkill to add all cases (6 or 7 of them). Also one might need a capitalized variant "Среда", "Декабрь".
This will further double the number of macro commands.

I think we can provide just a single case (or, maybe a couple?) - most likely to be needed, and those who need other cases can just write a new macro for decoding numeric into textual. Practically it is more time consuming to find a
command one needs for his case than to just write a fresh one.